ceph.spec.in

```diff
@@ -215,6 +215,7 @@ BuildRequires:	libtool
 BuildRequires:	libxml2-devel
 BuildRequires:	make
 BuildRequires:	ncurses-devel
+BuildRequires:	ninja-build
```

Comment:
I think it is just `ninja` on SuSE, so this needs to be moved to the "distro-conditional dependencies" section and bifurcated appropriately.